examples/node/lib/rest/api/v2010/account.ts

```diff
@@ -201,32 +201,34 @@ export class AccountContextImpl implements AccountContext {
   }
 
   remove(callback?: any): Promise<boolean> {
-    let operationVersion = this._version,
+    const instance = this;
+    let operationVersion = instance._version,
```

Comment:
Are we changing from `this` to `instance` for readability?

Reply:
It's to simplify the templates where we can assume `instance` exists in all cases.